### PR TITLE
update helm to 3.2.4 / alpine to 3.12

### PR DESCRIPTION
**Pull request by cpanato**
**What this PR does / why we need it**:
- update HELM_VERSION to v3.2.4
- alpine to 3.12

in the dockerfile:
- update python to use version 3

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by rimusz** [comment]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -1,14 +1,14 @@
-FROM alpine:3.11
+FROM alpine:3.12
 
 RUN apk --no-cache add \
     curl \
     git \
     libc6-compat \
     openssh-client \
-    python \
+    python3 \
```
Comment (unguiculus) [inline Dockerfile:8]: Se my comment at https://github.com/helm/chart-testing/pull/233/files#r441595835

**Comment by smlx** [comment]
Hi, just wanted to check on the status of this PR.

I'm hitting problems with the old version of helm in the chart-testing image that the chart-testing action uses by default. Specifically, the `lookup` function is failing validation (https://github.com/helm/helm/issues/7955). This is fixed in helm 3.2.x.

**Comment by unguiculus** [comment]
Closing this one in favor of #253.

**Pull request closed without merging**